lib/src/base-styles/_nx-page-header.scss

```diff
@@ -41,61 +41,60 @@ $main-header-product-grid-rows: $main-header-branding-height / 2;
   color: $nx-off-white;
   display: -ms-grid;
   display: grid;
-  grid-column-gap: 2px;
-  grid-row-gap: 2px;
-  -ms-grid-rows: $main-header-product-grid-rows 2px auto;
-  grid-template-rows: $main-header-product-grid-rows auto;
-  -ms-grid-columns: $main-header-branding-height 2px auto;
-  grid-template-columns: $main-header-branding-height auto;
+  grid-column-gap: $nx-spacing-xxs;
+  -ms-grid-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
+  grid-template-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
+  -ms-grid-columns: $main-header-product-height $nx-spacing-xxs auto;
+  grid-template-columns: $main-header-product-height auto;
   grid-template-areas:
     "logo title"
     "logo version";
-  height: $main-header-branding-height;
+  height: $main-header-product-height;
   margin-top: 10px;
   padding-right: 12px;
```

Comment:
Is this right? Should it be margin instead?

Reply:
> Is this right? Should it be margin instead?

Actually it's unnecessary: https://github.com/sonatype/sonatype-react-shared-components/pull/13/commits/c0c87699a0ea1b85f99c8cef03317a178d653ef1

Reply:
> Actually it's unnecessary: c0c8769

Are you sure?  In an application with buttons to the right of this (like IQ), what maintains the space between the product name and the buttons?

Reply:
> > Actually it's unnecessary: [c0c8769](https://github.com/sonatype/sonatype-react-shared-components/commit/c0c87699a0ea1b85f99c8cef03317a178d653ef1)
> 
> Are you sure? In an application with buttons to the right of this (like IQ), what maintains the space between the product name and the buttons?

Well in IQ nothing does, there is no right margin or padding present on the product block (or its children).

I don't have any issue with having any elements to the right of the product block using left margin for spacing. That seems more intentional than 12px of right margin here "just in case" even if it is zeroed out by `container-horizontal`.